snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -95,15 +95,15 @@ public void CreateBucket()
         {
             var projectId = _fixture.ProjectId;
 
-            // <Buckets.Insert>
+            // <Service.Buckets.Insert>
```

Comment:
I’d like us to find a way to show even the ugly snippets because even they’re WAY better than none at all.


On May 7, 2016, at 9:06 AM, Jon Skeet <notifications@github.com<mailto:notifications@github.com>> wrote:


In snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs<https://github.com/GoogleCloudPlatform/gcloud-dotnet/pull/141#discussion_r62418274>:

> @@ -95,15 +95,15 @@ public void CreateBucket()
>          {
>              var projectId = _fixture.ProjectId;
>
> -            // <Buckets.Insert>
> +            // <Service.Buckets.Insert>


Don't worry about this for the moment - basically the snippets are only likely to be meaningful within StorageClient. But might as well leave the commit in place.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/GoogleCloudPlatform/gcloud-dotnet/pull/141/files/9b3f60a6a7c089fa672b86b0cc93f81d09178459..6b6522308a2e56861953bea7b5fc4e601db0b969#r62418274>